vyper/venom/passes/simplify_cfg.py

```diff
@@ -85,7 +85,7 @@ def _collapse_chained_blocks(self, entry: IRBasicBlock):
     def _run_pass(self, ctx: IRFunction, entry: IRBasicBlock) -> None:
         self.ctx = ctx
 
-        for _ in range(len(ctx.basic_blocks)): # essentialy `while True`
+        for _ in range(len(ctx.basic_blocks)):  # essentialy `while True`
```

Comment:
i think we need an `else: raise CompilerPanic` to check we did not overflow the range bound

Reply:
right I forgot about that :/